### ISSUE TITLE
Support for Nix package manager

### ISSUE THREAD

**Issue by jnoortheen** [kiteco/issue-tracker#273]
It would be great if this can be installed with [nix](https://nixos.org/) package manager. Currently [there is no package](https://github.com/NixOS/nixpkgs/issues/91842) for  `kite ` . It is better to support to this since it can help install kite from multiple platforms with ease. 

There are two ways to do this, 
1. create a derivation and PR to nixpkgs 
2. create a flake and derivation and host in this repo itself
 - I prefer this way, so it can be part of the release cycle from github.

**Comment by metalogical** [comment]
We don't currently plan on supporting Nix, but you can try to create a derivation yourself with patchelf. If you manage to get something working, definitely let us know.

**Comment by jnoortheen** [comment]
Yes I will try creating one this weekend and will post here if got it working.
